HomePage.html

```diff
@@ -284,48 +309,63 @@ <h3 class="skill-card-heading">Languages</h3>
 
 	</section>
 
-	<section class="contact-form-page">
+	<section class="contact-form-page" id="contact">
+		
+		<div class="form-div">
+
 		<h3 class="form-title">I'm always interested in hearing about new projects, so if you'd like to chat please get in
 			touch.</h3>
 		
 			<form action="https://formspree.io/f/mgeqkdqj" method="post">
-			<input type="text" class="form-field" name="name" id="name" placeholder="Full name" required maxlength="30" >
-			<input type="email" class="form-field" name="email" id="email" placeholder="Email Address" required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$">
-			<textarea name="message" id="message" cols="30" rows="10" placeholder="Enter text here" required maxlength="500"></textarea>
-			<input type="submit" class="form-button" value="Get in Touch"  >
-		</form>
-
+				<input 
+					type="text" 
+					class="form-field" 
+					name="name" 
+					id="name" 
+					placeholder="Full name" 
+					required maxlength="30" >
+				
+				<input
+					type="email" 
+					class="form-field"
+					name="email"
+					id="email" placeholder="Email Address" 
+					required pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,}$">
+			
+				<textarea 
+					name="message" 
+					id="message" 
+					cols="30" 
+					rows="10" 
+					placeholder="Enter text here" 
+					required 
+					maxlength="500"></textarea>
+				
+				<input type="submit" class="form-button" value="Get in Touch"  >
+			</form>
```

Comment:
[REQUIRED] Good job implementing the changes requested by the previous reviewer but there is just a little more. Once you are done your project will be approved. You introduced some issues while implementing some of the changes.  Kindly take note of the numbers in the image below

|Your Design|
|------------|
|![image](https://user-images.githubusercontent.com/62938735/193400719-a12df50c-b457-43d6-a7f3-2d514996a788.png)|

- [ ] For numbers 1 and 4 the background image comes under some of the main content. So. the text on the left is on top of the image and the standing stick background is under some of the form inputs. I think what you can do is add container tags before putting your form and text inside them. So one container is the main container for that section and you can add your background image to that while the second container wraps the text and form. directly and you can just give a smaller width like maybe 80% of the whole container. This way the form. and text will never extend to the entire width and will be kept separate from the background image. on. the left and right. 
- [ ] For number 2 the background image on the right for your mobile screen is showing in. your desktop screen. Kindly remove that
- [ ] For number 3 you added a box to the footer. If you look at the design template you will notice that it is only a faint line at the top. You can use border at the top only or hr tag for this